src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/repository/EventAuditRepository.kt

```diff
@@ -32,11 +32,12 @@ interface EventAuditRepository : JpaRepository<EventAudit, Long> {
   @Transactional
   @Modifying
   @Query(
-    "Update event_audit SET application_method_type=:#{#applicationMethodType.name()}  " +
+    "Update event_audit SET application_method_type=:#{#applicationMethodType.name()},  " +
```

Comment:
would this be an insert everytime instead of update?

Reply:
narr this is to update the application method type at the end of the process.